data/macos.yaml

```diff
@@ -292,6 +315,19 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#Kernel_Extension'
 ---
+name: MacOSDuetKnowledgeBase
+doc: KnowledgeC User and Application usage database
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/Library/Application Support/Knowledge/knowledgeC.db'
+    - '/private/var/db/CoreDuet/Knowledge/knowledgeC.db'
+labels: [Users, Logs]
+supported_os: [Darwin]
+urls:
+- 'https://www.mac4n6.com/blog/2018/8/5/knowledge-is-power-using-the-knowledgecdb-database-on-macos-and-ios-to-determine-precise-user-and-application-usage'
```

Comment:
seeing this is a single url please change to `urls: ['https://...']`

Reply:
fixed